app/views/merchants/_merchants_index.html.erb

```diff
@@ -2,7 +2,7 @@
 <p align="center"><%= link_to "New Merchant", "/merchants/new" %></p>
 <section class = "grid-container">
   <% @merchants.each do |merchant|%>
-    <section class = "grid-item" id = <%= "merchant_#{merchant.id}"%>>
+    <section id = <%= "merchant_#{merchant.id}"%>>
```

Comment:
This class was pulling in CSS that made this render in an unappealing way.

Reply:
lol. It really did